files/home/mastodon/live/lib/tasks/digital_ocean.rake

```diff
@@ -40,7 +40,40 @@ namespace :digitalocean do
       env['REDIS_PORT'] = 6379
 
       if prompt.yes?('Do you want to store user-uploaded files on the cloud?', default: false)
-        case prompt.select('Provider', ['Amazon S3', 'Wasabi', 'Minio', 'Google Cloud Storage'])
+        case prompt.select('Provider', ['DigitalOcean Space','Amazon S3', 'Wasabi', 'Minio', 'Google Cloud Storage'])
+        when 'DigitalOcean Space'
+          env['S3_ENABLED'] = 'true'
+          env['S3_PROTOCOL'] = 'https'
+
+          env['S3_BUCKET'] = prompt.ask('Space name:') do |q|
+            q.required true
+            q.default "files.#{env['LOCAL_DOMAIN']}"
+            q.modify :strip
+          end
+
+          env['S3_REGION'] = prompt.ask('Space region:') do |q|
+            q.required true
+            q.default 'nyc3'
+            q.modify :strip
+          end
+
+          env['S3_HOSTNAME'] = prompt.ask('Space endpoint:') do |q|
```

Comment:
If they include https:// I think this will break down below

Reply:
Why should they? Prompt says "Space endpoint" and DigitalOcean clearly provides endpoints without protocol

Reply:
fair enough